allennlp/modules/elmo.py

```diff
@@ -483,8 +512,12 @@ def forward(self,  # pylint: disable=arguments-differ
         """
         Parameters
         ----------
-        inputs: ``torch.Tensor``
-            Shape ``(batch_size, timesteps, 50)`` of character ids representing the current batch.
+        inputs: ``torch.Tensor``, required.
+            The shape depends on whether you have cached character convolutions.
```

Comment:
Ah, great, here we get some documentation about this.  This is a pretty important point, so I'd definitely mention it everywhere that `vocab_to_cache` appears.  Also, looks like this will crash if you don't cache the whole vocab?  Yes?  That'd be good to mention also.